platform/web/os_web.h

```diff
@@ -73,6 +73,8 @@ class OS_Web : public OS_Unix {
 	// Override return type to make writing static callbacks less tedious.
 	static OS_Web *get_singleton();
 
+	_FORCE_INLINE_ bool is_memory64() const { return sizeof(size_t) == sizeof(uint64_t); }
```

Comment:
Is this `true` or `false` with `wasm32+64bitptrs`?

Reply:
> Is this `true` or `false` with `wasm32+64bitptrs`?

This is `true`. The previous `WASM_MEMORY64_ENABLED` define was applied to `wasm32+64bitptrs`.

Reply:
> > Is this `true` or `false` with `wasm32+64bitptrs`?
> 
> This is `true`. The runtime is `wasm32`, but the pointer sizes are 64-bit wide. The previous `WASM_MEMORY64_ENABLED` define was applied to `wasm32+64bitptrs`.

This is why I specified the option as `+64bitptrs` instead of choosing a simpler, but way more cryptic `wasm32+64` (which would imply `wasm32+wasm64`, which is not the case).

Reply:
Mmm. Makes me reconsider the use of actual name of the function `is_memory64()`, as it may be misleading. Technically, `wasm32+64bitptrs` isn't truly [MEMORY64](https://webassembly.org/features/#table-row-memory64).